hal/randombytes.h

```diff
@@ -0,0 +1,27 @@
+#ifndef PQCLEAN_RANDOMBYTES_H
+#define PQCLEAN_RANDOMBYTES_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#ifdef _WIN32
+/* Load size_t on windows */
+#include <crtdefs.h>
+#else
+#include <unistd.h>
+#endif /* _WIN32 */
+
+/*
+ * Write `n` bytes of high quality random bytes to `buf`
+ */
+#define randombytes     PQCLEAN_randombytes
+int randombytes(uint8_t *output, size_t n);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* PQCLEAN_RANDOMBYTES_H */
```

Comment:
We don't need the namespacing here and we don't need to support C++ or Windows.
You can simplify to: 
```suggestion
// SPDX-License-Identifier: Apache-2.0
#ifndef RANDOMBYTES_H
#define RANDOMBYTES_H

#include <stdint.h>
#include <unistd.h>

int randombytes(uint8_t *output, size_t n);

#endif /* RANDOMBYTES_H */
```